vms/platformvm/config/execution_config_test.go

```diff
@@ -39,6 +41,18 @@ func TestExecutionConfigUnmarshal(t *testing.T) {
 	t.Run("all values extracted from json", func(t *testing.T) {
 		require := require.New(t)
 		b := []byte(`{
+			"network": {
+				"max-validator-set-staleness": 1,
+				"target-gossip-size": 2,
+				"pull-gossip-poll-size": 3,
+				"pull-gossip-frequency": 4,
+				"pull-gossip-throttling-period": 5,
+				"pull-gossip-throttling-limit": 6,
+				"expected-bloom-filter-elements":7,
+				"expected-bloom-filter-false-positive-probability": 8,
+				"max-bloom-filter-false-positive-probability": 9,
+				"legacy-push-gossip-cache-size": 10
```

Comment:
Can we test that the defaults get populated correctly if only some of the network config is provided? I know it should... but it seems like an interesting enough of a test